src/components/Header.js

```diff
@@ -63,7 +63,7 @@ const Header = class extends React.Component {
        console.log(bulmaCollapsibleInstance.collapsed());
    });
 
-   const bulmaCollapsibleElement_1 = new bulmaCollapsible('#collapsible-message-accordion-1');
+  /* const bulmaCollapsibleElement_1 = new bulmaCollapsible('#collapsible-message-accordion-1');
```

Comment:
excluding this piece of code, make i t works. I dont receive anymore this error:

```javascript
index.js:2177 TypeError: Cannot read property 'container' of undefined
    at new Component (component.js:10)
    at new bulmaCollapsible (index.js:9)
    at Header.componentDidMount (Header.js:66)
    at Header.componentDidMount (react-hot-loader.development.js:654)
    at commitLifeCycles (react-dom.development.js:17335)
    at commitAllLifeCycles (react-dom.development.js:18737)
    at HTMLUnknownElement.callCallback (react-dom.development.js:150)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:200)
    at invokeGuardedCallback (react-dom.development.js:257)
    at commitRoot (react-dom.development.js:18949)
    at react-dom.development.js:20419
    at Object.unstable_runWithPriority (scheduler.development.js:255)
    at completeRoot (react-dom.development.js:20418)
    at performWorkOnRoot (react-dom.development.js:20347)
    at performWork (react-dom.development.js:20255)
    at performSyncWork (react-dom.development.js:20229)
    at requestWork (react-dom.development.js:20098)
    at scheduleWork (react-dom.development.js:19912)
    at scheduleRootUpdate (react-dom.development.js:20573)
    at updateContainerAtExpirationTime (react-dom.development.js:20601)
    at updateContainer (react-dom.development.js:20658)
    at ReactRoot../node_modules/@hot-loader/react-dom/cjs/react-dom.development.js.ReactRoot.render (react-dom.development.js:20954)
    at react-dom.development.js:21091
    at unbatchedUpdates (react-dom.development.js:20460)
    at legacyRenderSubtreeIntoContainer (react-dom.development.js:21087)
    at render (react-dom.development.js:21199)
    at app.js:62
__stack_frame_overlay_proxy_console__ @ index.js:2177
error @ react-hot-loader.development.js:244
componentDidCatch @ react-hot-loader.development.js:2160
callback @ react-dom.development.js:18096
callCallback @ react-dom.development.js:16434
commitUpdateEffects @ react-dom.development.js:16473
commitUpdateQueue @ react-dom.development.js:16461
commitLifeCycles @ react-dom.development.js:17365
commitAllLifeCycles @ react-dom.development.js:18737
callCallback @ react-dom.development.js:150
invokeGuardedCallbackDev @ react-dom.development.js:200
invokeGuardedCallback @ react-dom.development.js:257
commitRoot @ react-dom.development.js:18949
(anonymous) @ react-dom.development.js:20419
unstable_runWithPriority @ scheduler.development.js:255
completeRoot @ react-dom.development.js:20418
performWorkOnRoot @ react-dom.development.js:20347
performWork @ react-dom.development.js:20255
performSyncWork @ react-dom.development.js:20229
requestWork @ react-dom.development.js:20098
scheduleWork @ react-dom.development.js:19912
scheduleRootUpdate @ react-dom.development.js:20573
updateContainerAtExpirationTime @ react-dom.development.js:20601
updateContainer @ react-dom.development.js:20658
./node_modules/@hot-loader/react-dom/cjs/react-dom.development.js.ReactRoot.render @ react-dom.development.js:20954
(anonymous) @ react-dom.development.js:21091
unbatchedUpdates @ react-dom.development.js:20460
legacyRenderSubtreeIntoContainer @ react-dom.development.js:21087
render @ react-dom.development.js:21199
(anonymous) @ app.js:62
setTimeout (async)
(anonymous) @ ready.js:32
(anonymous) @ app.js:61
Promise.then (async)
(anonymous) @ app.js:54
Promise.then (async)
./.cache/app.js @ app.js:16
__webpack_require__ @ bootstrap:766
fn @ bootstrap:129
0 @ tags.js:98
__webpack_require__ @ bootstrap:766
(anonymous) @ bootstrap:903
(anonymous) @ bootstrap:903
index.js:2177 The above error occurred in the <AppContainer> component:
    in AppContainer (created by HotExportedHomePage)
    in HotExportedHomePage (created by PageRenderer)
    in PageRenderer (created by JSONStore)
    in JSONStore (created by EnsureResources)
    in ScrollContext (created by EnsureResources)
    in RouteUpdates (created by EnsureResources)
    in EnsureResources (created by RouteHandler)
    in RouteHandler (created by Root)
    in div (created by FocusHandlerImpl)
    in FocusHandlerImpl (created by Context.Consumer)
    in FocusHandler (created by RouterImpl)
    in RouterImpl (created by LocationProvider)
    in LocationProvider (created by Context.Consumer)
    in Location (created by Context.Consumer)
    in Router (created by Root)
    in Root
    in _default
```

React will try to recreate this component tree from scratch using the error boundary you provided, LocationProvider.

Probably it is not needed, and it is needed only this:

```js

   const bulmaCollapsibleInstances = bulmaCollapsible.attach('.is-collapsible');

```